ndmg/scripts/ndmg_bids.py

```diff
@@ -318,15 +283,9 @@ def main():
     parser.add_argument(
         "output_dir",
         help="The directory where the output "
-        "files should be stored.",
-    )
-    parser.add_argument(
-        "--modality",
-        help="Modality of MRI scans that"
-            " are being evaluated. Functional modality"
-            " analysis still under development.",
-        choices=["dwi", "func"],
-        default="dwi",
+        "files should be stored. If you are running group "
+        "level analysis this folder should be prepopulated "
```

Comment:
FYI: Ross removed group analysis in his branch too, which is why the help text was changed